src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkTask.java

```diff
@@ -24,16 +26,32 @@ This product includes software developed at Datadog (https://www.datadoghq.com/)
     DatadogLogsApiWriter writer;
     int remainingRetries;
 
+    // Only for testing
+    public Integer retryOverride;
+
     @Override
     public void start(Map<String, String> settings) {
         log.info("Starting Sink Task.");
         config = new DatadogLogsSinkConnectorConfig(settings);
         initWriter();
+        logMaxRetryBackoffMs(config.retryMax, config.retryBackoffMs);
```

Comment:
As `retryMax` and `retryBackoffMs` are configurable, I would add `MaxRetryTimeMs` in the configuration.
I would use `MaxRetryTimeMs` as the maximum waiting time and remove the warning.